### ISSUE TITLE
keyctl instantiate

### ISSUE THREAD

**Issue by mythi** [landhb/linux-keyutils#12]
I wanted to do the equivalent of `keyctl instantiate <key> <data> <keyring>` but I can't see how it's possible with the current functionality. 

Would  `instantiate()` method for `Key` make sense?

**Comment by landhb** [comment]
@mythi Definitely, I'd be willing to accept a PR if you'd like to submit one. Just need to add a method that uses `KeyCtlOperation::Instantiate`

Currently you can only create fully instantiated keys with this library, so I didn't implement any of the functions for the [`request_key` workflow](https://man7.org/linux/man-pages/man2/request_key.2.html)
